swagger.yaml

```diff
@@ -246,6 +246,55 @@ paths:
           description: Wrong content type (should be plain text)
         422:
           description: Missing fields
+  /projects/{projectId}/collaboration:
+    put:
+      tags:
+        - projects
+        - collaboration
+      summary: Creates an invite token for project sharing
+      security:
+        - user_id: []
+      requestBody:
+        conent:
+          application/json:
+            schema:
+              type: object
+      description: Creates a valid invite token
+      responses:
+        201:
+          description: Successfully created an invite token
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/InviteToken"
+        400:
+          description: Malformed Request
+        401:
+          description: Unauthorized
+        404:
+          description: No project found
```

Comment:
Probably 403 will be needed as well

Reply:
Can you distinct the usage of 403 from 401 in that case, because i don't quite see which one fits in the right situation

Reply:
401 - user didn't provide data for authentication

403 - user has no right to access resource

Reply:
alright, that makes sense